.travis.yml

```diff
@@ -0,0 +1,16 @@
+sudo: false # Explicitly use container-based infrastructure
+
+language: go
+
+go:
+    - 1.5
+
+before_install:
+- wget https://downloads.giantswarm.io/builder/0.11.0/builder
+- chmod +x ./builder
+- export PATH=$PATH:$PWD
```

Comment:
Builder is currently not used. Do you want to use it later for pinning versions?